yarn.lock

```diff
@@ -9367,6 +9367,7 @@ __metadata:
     react-hook-form: 7.45.2
     typescript: ^5.1.6
     viem: ^2.22.17
+    vitest: ^3.0.4
```

Comment:
There are 2 versions used in this project: `^3.0.4` and `^2.1.9`.

Reply:
Thanks for flagging. There was a CVE in vitest. I'll update in a future PR today